Rules/CTF/Scripts/CTF_GiveSpawnItems.as

```diff
@@ -9,7 +9,7 @@ const u32 materials_wait_warmup = 40; //seconds between free mats
 //property
 const string SPAWN_ITEMS_TIMER = "CTF SpawnItems:";
 
-string base_name() { return "tent"; }
+const string[] base_name = {"tent", "warboat", "ballista"};
```

Comment:
Const variables should be written in uppercase. Don't forget to rename all occurrences of this variable.
```suggestion
const string[] BASE_NAME = {"tent", "warboat", "ballista"};
```

Reply:
I would avoid changing anything that isn't broken unless you are sure of the change 😁